config/dev.exs

```diff
@@ -73,3 +73,5 @@ config :phoaw, Phoaw.Repo,
   database: "phoaw_dev",
   hostname: "localhost",
   pool_size: 10
+
+  import_config "dev.secret.exs"
```

Comment:
is this indentation right? Are we possible to override non db config?

Reply:
all config is possible to override.